WTWSMS/decisions/WtWSMS_realm_decisions.txt

```diff
@@ -1140,4 +1140,41 @@ decisions = {
 			}
 		}
 	}
+	divide_roman_empire = {
+		only_playable = yes
+		is_high_prio = yes
+		potential = {
+			has_landed_title = e_roman_empire
+			age = 16
+			realm_size = 200
+			tier = emperor
+		}
+		allow = {
+			scaled_wealth = 1.0
+			prisoner = no
+			war = no
+			NOT = { trait = incapable }
+			any_dynasty_member = { is_heir = yes }
+			completely_controls = c_byzantion
+			completely_controls = c_roma
+			completely_controls = c_ravenna
+		}
+		effect = {
+			custom_tooltip = {
+				text = divide_roman_empire_tooltip
+				narrative_event = { id = imperial_legacy.50 }
+			}
+		}
+		ai_will_do = {
+			factor = 1
+			modifier = {
+				factor = 3
+				trait = lunatic
+			}
+			modifier = {
+				factor = 0
+				trait = ambitious
+			}
```

Comment:
The AI will take this far too frequently with these modifiers.

Reply:
It's still the old AI code I keep forgetting to fix :S

Reply:
Should be much better now